Resources/Prototypes/Guidebook/med.yml

```
@@ -0,0 +1,8 @@
+- type: guideEntry
+  id: Chemistry
+  name: guide-entry-chemistry
+<<<<<<< HEAD
+  text: "/Server Info/Guidebook/Med/Chemistry.xml"
+=======
+  text: "/Server Info/Guidebook/Med/Chemistry.xml"
+>>>>>>> ac73b1ab677079ccda328ec66ab6cc65f171ff5f
```

```suggestion
  text: "/Server Info/Guidebook/Med/Chemistry.xml"
```
More here, should be just a single line